beartype/_util/py/utilpyversion.py

```diff
@@ -103,6 +103,14 @@
 '''
 
 
+#FIXME: After dropping Python 3.7 support, *REMOVE* all code conditionally
+#testing this global.
+IS_PYTHON_3_7 = version_info[:2] == (3, 7)
```

Comment:
This is new. I added this because I'm a luddite who couldn't keep things readable (at least not to myself) without it or something similar.